Library/Styles/BaseStyles.swift

```diff
@@ -141,6 +142,14 @@ public let separatorStyle: ViewStyle = { (view: UIView) in
     |> \.accessibilityElementsHidden .~ true
 }
 
+public let greenCTAButtonStyle: ButtonStyle = { (button: UIButton) in
+  button
+    |> roundedStyle(cornerRadius: 12)
+    |> UIButton.lens.backgroundColor(for: .normal) .~ .ksr_green_500
+    |> UIButton.lens.backgroundColor(for: .highlighted) .~ .ksr_green_500
```

Comment:
Ah, I think this is why the button doesn't seem to highlight when selected/highlighted..we're using the same color 🤔

Reply:
Now using `greenButtonStyle` from `ButtonStyles.swift` which has highlighted state handling 👍